molecularnodes/io/wwpdb.py

```diff
@@ -91,9 +92,14 @@ def execute(self, context):
         if scene.MN_import_node_setup:
             style = scene.MN_import_style
 
+        if not scene.MN_import_centre:
```

Comment:
same thing that we don't need to check if this exists as blender won't let it be `None`

Reply:
Done.

Reply:
Sorry this was a mistake on my part. This is what is currently now breaking all of the tests. I'll fix it and then explain in another comment.